en-US/downloads/gnulinux-dl.md

```diff
@@ -5,45 +5,33 @@ title: FontForge on GNU+Linux and other UNIXes
 archive_extension: .tar.gz|.zip
 ---
 
-Most of the GNU+Linux distributions have packages for fontforge, but these will often be a quite old.
+Most of the GNU+Linux distributions have packages for fontforge, but these will often be a quite old. If you want the latest FontForge, these are your options:
 
 ### AppImage
 
-Use the precompiled AppImage: [2020-03-14](https://github.com/fontforge/fontforge/releases/download/20200314/FontForge-2020-03-14-67687b0-x86_64.AppImage)
+AppImage is a binary format that has improved to the point where it is generally usable by all GNU+Linux users.
 
-### Ubuntu
+With each release, we compile an AppImage. The latest is [2020-03-14](https://github.com/fontforge/fontforge/releases/download/20200314/FontForge-2020-03-14-67687b0-x86_64.AppImage).
 
-Installing on Ubuntu is easy with our [Ubuntu PPA](https://launchpad.net/%7Efontforge/+archive/ubuntu/fontforge)
+We recommend all users use this method to get the latest release if their distribution has not yet provided it. The Ubuntu PPA, and similar distribution-specific build platforms that put the onus on us, are deprecated and not to be used.
 
-```sh
-sudo add-apt-repository ppa:fontforge/fontforge;
-sudo apt-get update;
-sudo apt-get install fontforge;
-```
+### Compiling
 
-### Compiling from Github
+#### From GitHub
 
-Check out [Building From Source](../source) for more information on using the bleeding edge.
+Check out [Building From Source](../source), as well as [`INSTALL.md`](https://github.com/fontforge/fontforge/blob/master/INSTALL.md), for more information on using bleeding edge versions of FontForge, which change every time a developer merges a pull request.
 
-### Installing from an rpm
```

Comment:
I removed this as well as we have not provided RPM in quite a while, just as pointless in 2020 as PPA.